src/renderer/helpers/chainHelper.ts

```diff
@@ -26,6 +26,20 @@ export const isBaseChain = (chain: Chain): chain is typeof BASE_CHAIN => eqChain
  */
 export const isBaseChainAsset = ({ chain }: Asset): boolean => isBaseChain(chain)
 
+/**
+ * Check whether chain is a CROSS chain
+ */
+export const isCrossChain = (chain: Chain): boolean => !isBaseChain(chain)
+
+/**
+ * Check whether asset is a CROSS chain asset
+ */
+export const isCrossChainAsset = (asset: Asset): boolean => {
+  if (isBaseChainAsset(asset)) return false
+  const crossChainAsset = getChainAsset(asset.chain)
+  return eqAsset.equals(crossChainAsset, asset)
```

Comment:
im not sure here but should asset ALWAYS equals to the `getChainAsset(asset.chain)` ?

Reply:
Yes, we always have one cross chain asset for each chain. If an asset is equal to this "chain asset", then it's exactly this cross chain asset.